cpp/src/tsne/utils.cuh

```diff
@@ -47,17 +49,25 @@
  * @param[in] stream: The GPU stream.
  * @param[in] seed: If seed == -1, then the output is pure randomness. If >= 0, then you can reproduce TSNE.
  */
-void random_vector(float *vector, const float minimum, const float maximum,
-                   const int size, cudaStream_t stream, long long seed = -1) {
+struct isnan_test {
```

Comment:
We should consider adding this to a file for debugging utilities in raft. Maybe something like `debug_utils.cuh`